lib/convenient_service/support/dependency_container/entities/method.rb

```diff
@@ -24,29 +26,37 @@ class Method
           attr_reader :body
 
           ##
-          # @param full_name [String, Symbol]
+          # @!attribute [r] alias_slug
+          #   @return [String, Symbol]
+          #
+          attr_reader :alias_slug
+
+          ##
+          # @param slug [String, Symbol]
           # @param scope [:instance, :class]
           # @param body [Proc]
+          # @param alias_slug [String, Symbol]
           # @return [void]
           #
-          def initialize(full_name:, scope:, body:)
-            @full_name = full_name
+          def initialize(slug:, scope:, body:, alias_slug: nil)
```

Comment:
**Critical**

Please, use an empty string as the `alias_slug` default value. This way a lot of `nil` checks can be avoided.